src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final String DEFAULT_COLOR = "WHITE";
+    public static final int BOUND_MAX = 10000;
+    private Random random = new Random();

```

Comment:
could be `final` as well, please separate fields from constants with new line.

Reply:
ok, done